### PR TITLE
Test: add unit test for atom_in and cal_epsilon

### LINKED ISSUE #2404: I will add the UT for atom_in and cal_epsilon
### Describe Current Status and Possible Solution

UT for atom_in and cal_epsilon is lacking.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by sunml99**
Fix #2404

### CODE REVIEW

File: source/module_hamilt_general/module_surchem/test/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+list(APPEND depend_files 
+  ../../../module_base/global_variable.cpp
+)
+AddTest(
+  TARGET surchem_atom_in
+  SOURCES atom_in_test.cpp
+)
+
+AddTest(
+  TARGET surchem_cal_epsilon  LIBS ${math_libs} planewave device base
```
Comment (hongriTianqi) [inline source/module_hamilt_general/module_surchem/test/CMakeLists.txt]:
```suggestion
  TARGET surchem_cal_epsilon
  LIBS ${math_libs} planewave device

File: source/module_hamilt_general/module_surchem/test/cal_epsilon_test.cpp [hunk 2]
```diff
@@ -0,0 +1,135 @@
+#ifdef __MPI
+#include "mpi.h"
+#include "../../../module_basis/module_pw/test/test_tool.h"
```
Comment (hongriTianqi) [inline source/module_hamilt_general/module_surchem/test/cal_epsilon_test.cpp:3]: Do we need to include this header file?

Comment (sunml99) [reply source/module_hamilt_general/module_surchem/test/cal_epsilon_test.cpp:3]: Are you referring to “test_tool.h"? Indeed not necessary, and just delete it, thanks.

Comment (hongriTianqi) [reply source/module_hamilt_general/module_surchem/test/cal_epsilon_test.cpp:3]: Yes, I thinks it is unnecessary.

File: source/module_hamilt_general/module_surchem/test/CMakeLists.txt (hunk 1 above)
Comment (sunml99) [reply source/module_hamilt_general/module_surchem/test/CMakeLists.txt]: Perhaps the "base" item is missing.  Should be "LIBS ${math_libs} planewave device base"?

Comment (hongriTianqi) [reply source/module_hamilt_general/module_surchem/test/CMakeLists.txt]: Yes, please. It's my fault. I just wanted to separate the LIBS to a second line.

File: source/module_hamilt_general/module_surchem/test/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  ../../../module_base/global_variable.cpp
+)
+AddTest(
+  TARGET surchem_atom_in
+  SOURCES atom_in_test.cpp
+)
+
+AddTest(
+  TARGET surchem_cal_epsilon
+  LIBS ${math_libs} planewave device
```
Comment (hongriTianqi) [inline source/module_hamilt_general/module_surchem/test/CMakeLists.txt]:
```suggestion
  LIBS ${math_libs} planewave device base

**Review by hongriTianqi — APPROVED** [review]

**Pull request merged**